Source/Core/Core/HW/EXI_DeviceSlippi.cpp

```diff
@@ -2286,13 +2286,25 @@ void CEXISlippi::prepareOnlineMatchState()
 		}
 
 		// Overwrite local player character
-		onlineMatchBlock[0x60 + (lps.playerIdx-1) * 0x24] = lps.characterId;
+		onlineMatchBlock[0x60 + (lps.playerIdx - 1) * 0x24] = lps.characterId;
```

Comment:
If you merge the doubles branch into yours, it should pull in the playerIdx changes so you don't need to subtract 1 here. I fixed it to be 0 indexed everywhere so it's not confusing anymore

Reply:
Done